engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -2190,8 +2192,12 @@ namespace dmGameSystem
 
     static dmGameObject::CreateResult CompGuiAddToUpdate(const dmGameObject::ComponentAddToUpdateParams& params) {
         GuiComponent* gui_component = (GuiComponent*)*params.m_UserData;
-        gui_component->m_AddedToUpdate = true;
-        return dmGameObject::CREATE_RESULT_OK;
+        if (gui_component->m_Initialized) {
+            gui_component->m_AddedToUpdate = true;
+            return dmGameObject::CREATE_RESULT_OK;
+        }
+        gui_component->m_AddedToUpdate = false;
```

Comment:
Isn't this false already?
I generally want us to avoid initializing the same value multiple times (every little code snippet adds up...).
Same could be asked with the "Initialized = 0" above. Presumably they were already 0?